cmd/minikube/cmd/node_delete.go

```diff
@@ -46,7 +46,7 @@ var nodeDeleteCmd = &cobra.Command{
 
 		err = node.Delete(*cc, name)
 		if err != nil {
-			out.FatalT("Failed to delete node {{.name}}", out.V{"name": name})
+			exit.WithError("deleting node", err)
```

Comment:
why not including the name?

Reply:
I just need exit here and out.FatalT doesn't that. I can change it something like exit.WithCodeT but it's not that important yet.